### DISCUSSION TITLE
pointer question

### DISCUSSION THREAD

**Discussion by k1attila1** [edubart/nelua-lang#153] [Q&A] [answered]
Is it possible in Nelua ?

int* pMemory = 0x12AED567;
*pMemory = 2345;

thank you

**Comment by edubart** [comment, marked as answer]
> int* pMemory = 0x12AED567; *pMemory = 2345;

Yes, like:

```lua
local pMemory: *cint = (@*cint)(0x12AED567)
$pMemory = 2345
```
